src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -124,26 +124,26 @@ function adjustStudentsPanelView(panelHeading, panelCollapse) {
 function expandCollapseExistingStudentsPanel() {
     const panelHeading = $(this);
     const panelCollapse = panelHeading.parent().children('.panel-collapse');
-    const displayStatus = panelHeading.children('.display-status');
-    const ajaxPreloadImage = displayStatus.children('#ajax-preload-image');
-    const ajaxStatusMessage = displayStatus.children('.ajax-status-message');
+
+    const ajaxPreloadImage = panelHeading.parent().find('#ajax-preload-image');
+    const ajaxStatusMessage = panelHeading.parent().find('.ajax-status-message');
     ajaxStatusMessage.hide(); // hide any status message from the previous state
 
     // perform AJAX only if existing students' spreadsheet is empty
     if (getSpreadsheetLength(dataHandsontable.getData()) === 0) {
         getAjaxStudentList(ajaxPreloadImage)
                 .then((data) => {
-                    ajaxPreloadImage.hide();
+                    ajaxPreloadImage.toggle();
```

Comment:
Actually around here are not good places to use `toggle`. The logic is clearer if you make it explicitly show or hide. Also, if there's some error and your initial state is wrong, then all the show/hide will be wrong.

Good places to use `toggle` is when you have something like `if (isVisible) hide() else show()`